app/models/group.rb

```diff
@@ -1,2 +1,5 @@
 class Group < ApplicationRecord
+  belongs_to :user, foreign_key: 'owner_id'
```

Comment:
Hey @wfbraga,

I think it would be better to call this relation by `owner`, to get the group's owner with `group.owner` instead of `group.user`.

Specially because we are going to create `group.users` in the future so it can be confusing.

What do you think?

Reply:
I agree.
I'll make the changes from
`belongs_to :user`
to
`belongs_to :owner`